setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup
+
+
+__version__ = (0, 0, 1)
+
+
+setup(
+    name="nchelpers",
+    description="Helper classes and methods for Climate and Forecast NetCDF datasets",
+    keywords="NetCDF climate forecast",
+    packages=['nchelpers'],
+    version='.'.join(str(d) for d in __version__),
+    url="http://www.pacificclimate.org/",
+    author="Rod Glover",
```

Comment:
You might want to include a `CONTRIBUTORS` file and add myself, Basil Veerman and David Bronaugh, since we all also contributed either code or ideas contained in this code.